.github/workflows/deploy.yml

```diff
@@ -79,7 +79,7 @@ jobs:
 
       - name: create conda envs
         uses: ./docs/.github/actions/create-conda-envs
-        continue-on-error: ${{ github.event.inputs.skip_conda_environment_check == 'true' }}
+        if: ${{ ! inputs.skip_conda_environment_check }}
```

Comment:
(Don't need to use `== 'true'` anymore as https://github.com/actions/runner/issues/1483 has been closed)